### DISCUSSION TITLE
Any document on nr-cli

### DISCUSSION THREAD

**Discussion by panyogesh** [aligungr/UERANSIM#291] [Q&A]
Wanted to have controlled messaging using nr-cli.

**Comment by aligungr** [comment]
I think I could not fully understand you, but please see [here](https://github.com/aligungr/UERANSIM/wiki/Usage#usage-of-the-command-line-interface-cli) for Usage instructions for `nr-cli`.

**Comment by panyogesh** [comment]
What I meant was can we have some way to trigger registration and de-registration process (controlled via CLI).

**Reply by aligungr** [reply]
Registration procedure always controlled automatically, but you can trigger de-registration procedures from CLI.
For example you can execute the following:

```
nr-cli imsi-001010000000001
deregister normal
```

And by the way, to see the available commands, run `commands` command:

```
nr-cli imsi-001010000000001
commands
```

etc.
